hls-plugin-api/src/Ide/Types.hs

```diff
@@ -404,7 +412,9 @@ instance PluginMethod Request Method_TextDocumentCodeAction where
       uri = msgParams ^. L.textDocument . L.uri
 
 instance PluginMethod Request Method_CodeActionResolve where
-  pluginEnabled _ msgParams pluginDesc config = pluginEnabledConfig plcCodeActionsOn (configForPlugin config pluginDesc)
+  pluginEnabled _ msgParams pluginDesc config =
+    pluginResolverResponsible (msgParams ^. L.data_) pluginDesc
```

Comment:
An issue with this is that it assumes that all the resolve implementations are using `OwnedResolveData`, which they might not be.